services/wfm/src/test/java/org/openkilda/wfm/topology/event/service/IslServiceTest.java

```diff
@@ -0,0 +1,183 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.event.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import org.openkilda.model.Isl;
+import org.openkilda.model.IslStatus;
+import org.openkilda.model.LinkProps;
+import org.openkilda.model.Switch;
+import org.openkilda.model.SwitchId;
+import org.openkilda.model.SwitchStatus;
+import org.openkilda.persistence.PersistenceManager;
+import org.openkilda.persistence.TransactionManager;
+import org.openkilda.persistence.repositories.IslRepository;
+import org.openkilda.persistence.repositories.LinkPropsRepository;
+import org.openkilda.persistence.repositories.RepositoryFactory;
+import org.openkilda.persistence.repositories.impl.Neo4jSessionFactory;
+import org.openkilda.persistence.spi.PersistenceProvider;
+import org.openkilda.wfm.EmbeddedNeo4jDatabase;
+import org.openkilda.wfm.LaunchEnvironment;
+import org.openkilda.wfm.StableAbstractStormTest;
+import org.openkilda.wfm.config.provider.ConfigurationProvider;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+public class IslServiceTest extends StableAbstractStormTest {
```

Comment:
Can we don't use StableAbstractStormTest here? We should stop writing unit tests in that way, there is no need to setup kafka and storm cluster

Reply:
Yes, I fixed it.